### ISSUE TITLE
Why are you not using GitHub properly?

### ISSUE THREAD

**Issue by MikeLund** [hyt47/SteamDesktopAuthenticator-Mod-47#11]
Why do you just use it as a "website" instead of using the actual code control features?

I know you have modified SteamAuth, and the original SDA project. So why not make them a proper fork, and include these forks?

If you want to supply compiled binaries, that's fine. But don't make it be the only solution; a "source code" archive of over 120 megabytes is nonsense!

tl;dr please fork the dependencies, and reference them in your project. Thanks.

**Comment by hyt47** [comment]
I don't have time to update the files, sorry!
